content/spaces/locations.textile

```diff
@@ -80,7 +80,7 @@ The following are the properties of a location event payload:
 | member.clientId | The "client identifier":/auth/identified-clients for the member. | String |
 | member.connectionId | The unique identifier of the member's "connection":/connect. | String |
 | member.isConnected | Whether the member is connected to Ably or not. | Boolean |
-| member.lastEvent.name | The most recent "event":/spaces/avatar emitted by the member. Will be one of @enter@, @update@, @leave@ or @remove@. | String |
+| member.lastEvent.name | The most recent "event":/spaces/avatar emitted by the member. Will be one of @enter@, @update@, @updateProfile@, @leave@ or @remove@. | String |
```

Comment:
As above, this is actually wrong - the `name` here is of the presence event. Perhaps we should re-consider this whole object :/

Reply:
Just to check here - is `remove` not valid either, as that's a Spaces event, not presence?